tools/c7n_azure/tests_azure/tests_resources/test_cosmos_db.py

```diff
@@ -355,16 +372,16 @@ def test_set_ip_range_filter_append(self, update_mock):
         self.assertEqual(1, len(update_mock.mock_calls))
         name, args, kwargs = update_mock.mock_calls[0]
 
+        expected = set(['11.12.13.14', '21.22.23.24', get_ext_ip()])
+        expected.update(get_portal_ips())
```

Comment:
so its still unclear to me how this is supposed to work, with get_ext_ip.. your recording it on your machine, when we go to replay on a different, extip will be different and this would fail?, this seems like it needs a differential to not verify the full list and not use getextip when doing replay mode. maybe this would be a faster discussion in chat.

Reply:
this call is covered by vcr and is in the replay